src/quo2/screens/sheets/bottom_sheet.cljs

```diff
@@ -0,0 +1,70 @@
+(ns quo2.screens.sheets.bottom-sheet
+  (:require
+   [quo.previews.preview :as preview]
+   [quo.react-native :as rn]
+   [quo2.components.bottom-sheet.view :as bottom-sheet]
+   [quo2.components.markdown.text :as text]
+   [reagent.core :as reagent]
+   [quo2.foundations.colors :as colors]))
+
+(def descriptor [{:label "Show handle:"
+                  :key   :show-handle?
+                  :type  :boolean}
+                 {:label "Backdrop dismiss:"
+                  :key   :backdrop-dismiss?
+                  :type  :boolean}
+                 {:label "Disable drag:"
+                  :key   :disable-drag?
+                  :type  :boolean}
+                 {:label "Android back cancel:"
+                  :key   :back-button-cancel
+                  :type  :boolean}
+                 {:label "Scrollable:"
+                  :key   :scrollable
+                  :type  :boolean}])
+
+(defn cool-preview []
+  (let [state      (reagent/atom {:show-handle?       true
+                                  :backdrop-dismiss?  true
+                                  :disable-drag?      false
+                                  :back-button-cancel true})
+        visible    (reagent/atom false)
+        scrollable (reagent/cursor state [:scrollable])]
+    (fn []
+      [rn/view {:margin-bottom 50
+                :padding       16}
+       [preview/customizer state descriptor]
+       [:<>
+        [rn/view {:style {:align-items :center
+                          :padding     16}}
+         [rn/touchable-opacity {:on-press #(reset! visible true)}
+          [rn/view {:style {:padding-horizontal 16
+                            :padding-vertical   8
+                            :border-radius      4
+                            :background-color   (colors/primary-50)}}
+           [text/text {:color :secondary-inverse}
+            (str "Open sheet: " @visible)]]]]
+
+        [bottom-sheet/bottom-sheet (merge @state
+                                          {:visible?  @visible
+                                           :on-cancel #(reset! visible false)})
+         [rn/view {:style {:height          (if @scrollable 1200 400)
+                           :justify-content :center
+                           :align-items     :center}}
+          [rn/touchable-opacity {:on-press #(reset! visible false)}
+           [text/text {:color :link} "Close"]]
```

Comment:
Can we please use internationalization with this ?
`i18n` lib and you can call `clojure.string/capitalize` ?